ops/envs/geth.env

```diff
@@ -20,12 +20,12 @@ WS_PORT=8546
 WS_API=eth,net,rollup,web3
 WS_ORIGINS=*
 
-CHAIN_ID=420
+CHAIN_ID=987
```

Comment:
The reason you had to change the test chain ID was because Goerli uses 420 & the system addresses in Goerli are not the same as the system addresses that we want to use for our itests - that right?

Reply:
Correct!